test/integration/concierge_impersonation_proxy_test.go

```diff
@@ -1793,6 +1793,7 @@ func TestImpersonationProxy(t *testing.T) { //nolint:gocyclo // yeah, it's compl
 			"external-tls-cert-secret-name",
 			corev1.SecretTypeTLS,
 			map[string]string{
+				"ca.crt":            string(externallyProvidedTLSServingCertPEM),
```

Comment:
Maybe we should comment this, since you found that the `ca.crt` is not required, but it is typically provided by CertManager, and how it is optionally used instead of the other value on the actual CR (name escapes me atm).